docs/design/logs.md

```diff
@@ -345,7 +345,25 @@ only meant for OTel components itself and anyone writing extensions like custom
 Exporters etc.
 
 // TODO: Document the principles followed when selecting severity for internal
-logs // TODO: Document how this can cause circular loop and plans to address it.
+logs
+
+When OpenTelemetry components generate logs that could potentially feed back
+into OpenTelemetry, this can result in what is known as "telemetry-induced
+telemetry." To address this, OpenTelemetry provides a mechanism to suppress such
+telemetry using the `Context`. Components are expected to mark telemetry as
+suppressed within a specific `Context` by invoking
+`Context::enter_telemetry_suppressed_scope()`. The Logs SDK implementation
+checks this flag in the current `Context` and ignores logs if suppression is
+enabled.
+
+This mechanism relies on proper in-process propagation of the `Context`.
+However, external libraries like `hyper` and `tonic`, which are used by
+OpenTelemetry in its OTLP Exporters, do not propagate OpenTelemetry's `Context`.
+As a result, the suppression mechanism does not work out-of-the-box to suppress
+logs originating from these libraries.
+
+// TODO: Document how OTLP can solve this issue without asking external
```

Comment:
... do we know how to do this without context propagation?! Would love a TL;DR if so!

Reply:
We can't fix without context propagation. And context propagation is only needed if the export task moves around threads. What we can try to do is - create a Tokio_multi_thread_runtime outselves, set its thread_count=1, and hand that runtime to OTLPExporter, instead of the current way of it grabbing the current runtime. 
Once we know OTLP (more specifically tonic/reqwest) always run in a given thread, we can somehow suppress context in that thread once, and it should work forever then.

Obviously, this needs to be tried out!